### PR TITLE
Add GIGA R1 WiFi as compatible

### PR DESCRIPTION
**Pull request by Hannes7eicher**
This PR adds the Arduino Giga R1 WiFi as compatible board to in the comments to the following examples:
- [DirList.ino](https://github.com/arduino-libraries/Arduino_USBHostMbed5/blob/main/examples/DirList/DirList.ino)
- [FileRead.ino](https://github.com/arduino-libraries/Arduino_USBHostMbed5/blob/main/examples/FileRead/FileRead.ino)
- [FileWrite.ino](https://github.com/arduino-libraries/Arduino_USBHostMbed5/blob/main/examples/FileWrite/FileWrite.ino)

### CODE REVIEW

File: examples/DirList/DirList.ino [hunk 1]
```diff
@@ -5,7 +5,7 @@
   get a list of the existing folders and files.
 
   The circuit:
-   - Portenta H7
+   - Portenta H7 or Giga R1 WiFi
```
Comment (aliphys) [inline examples/DirList/DirList.ino]:
```suggestion
   - Portenta H7 or GIGA R1 WiFi
```

File: examples/FileRead/FileRead.ino [hunk 2]
```diff
@@ -7,7 +7,7 @@
   in your storage device and write some content inside.
 
   The circuit:
-   - Portenta H7
+   - Portenta H7 or Giga R1 WiFi
```
Comment (aliphys) [inline examples/FileRead/FileRead.ino]:
```suggestion
   - Portenta H7 or GIGA R1 WiFi
```

File: examples/FileWrite/FileWrite.ino [hunk 3]
```diff
@@ -5,7 +5,7 @@
   write a file, eventually overwriting the original content.
 
   The circuit:
-   - Portenta H7
+   - Portenta H7 or Giga R1 WiFi
```
Comment (aliphys) [inline examples/FileWrite/FileWrite.ino]:
```suggestion
   - Portenta H7 or GIGA R1 WiFi
```

**Comment by aliphys** [comment]
Do the examples function as expected when running on the GIGA R1 WiFi for you?

**Comment by Hannes7eicher** [comment]
@aliphys I can confirm I successfully tested the examples.

**Review by alrvid — APPROVED** [review]
Looks fine, considering that the author also confirms he has run the examples successfully

**Pull request merged**